_includes/form-wait.html

```diff
@@ -0,0 +1,19 @@
+<!-- Modal -->
+<div class="modal fade" id="status-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel">
+  <div class="modal-dialog" role="document">
+    <div class="modal-content">
+      <div class="modal-header">
+        <button type="button" class="close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">&times;</span></button>
+        <h4 class="modal-title" id="myModalLabel">Please Wait...</h4>
```

Comment:
how about we make this message a little more descriptive - something like "Creating your sandbox. Please wait."